README.md

```diff
@@ -13,7 +13,7 @@ Install via pip:
 pip install object_storage
 ```
 
-The current version is `0.13.3`.
```

Comment:
Since the README on master for the GitHub project is the most accessible documentation, it may be worthwhile putting the last Python 2.x release here as well (with an optional deprecation notice).